### DISCUSSION TITLE
Improve the discovery of submodules in the build script

### DISCUSSION THREAD

**Discussion by jonasraoni** [pkp/pkp-lib#7982] [Proposals]
**Describe the problem you would like to solve**
At this moment we have to create a commit containing the text `##github-username/branch-name##` to hint the build script where to look for updated submodules.

**Describe the solution you'd like**
Update the script `startSubmodulesTRAVIS.sh` to look for submodules/matching hashes automatically at the PR's origin.

**Who is asking for this feature?**
Developer

**Comment by henriqueramos** [comment]
Already made on #7125.

https://github.com/pkp/ojs/pull/3160/files#diff-8f16e27c2e43ab2ebfaa7bf22fb77c3be7e92671fc0d73cc33ef8c47517d0df5

**Comment by jonasraoni** [comment]
I created this issue to go one step forward, and avoid the need of creating a commit with the message `##github-username/branch-name##` =]
